packages/web/src/scss/components/Card/README.md

```diff
@@ -0,0 +1,398 @@
+
+In a typical use case, you will display multiple Cards in a [Grid][grid].
+
+```html
+<div class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
+  <article class="Card Card--vertical">
+    <!-- … -->
+  </article>
+  <article class="Card Card--vertical">
+    <!-- … -->
+  </article>
+  <article class="Card Card--vertical">
+    <!-- … -->
+  </article>
+</div>
+```
+
+Depending on your situation, you may want to use the list semantics. And it will work!
+
+```html
+<ul class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
+  <li class="Card Card--vertical">
+    <!-- … -->
+  </li>
+  <li class="Card Card--vertical">
+    <!-- … -->
+  </li>
+  <li class="Card Card--vertical">
+    <!-- … -->
+  </li>
+</ul>
+```
+
+## Best Practices
+
+### Making the Whole Card Clickable
+
+To make the whole Card clickable, use the provided CardLink subcomponent. For best accessibility, you would typically
+wrap your CardTitle text in the CardLink component:
+
+```html
+<h4 class="CardTitle">
+  <a href="#" class="CardLink">Card title</a>
+</h4>
+```
+
+This establishes a clickable overlay over the whole Card, making it easier for users to interact with the Card.
+
+ℹ️ Don't worry, any interactive elements inside the Card (like links or buttons) will still work as expected.
+
+ℹ️ The text content of CardBody remains selectable and copyable even when the whole Card is clickable.
+
+### Linking the Media
+
+In most cases, using just a single link in the CardTitle and
+[making the whole card clickable](#making-the-whole-card-clickable) is the best approach in terms of accessibility.
+The Card will have a single accessible link which will be announced by screen readers.
+
+However, if you cannot use the CardLink subcomponent, and you still need to make the media clickable, you can wrap the
+CardMedia image in a link:
+
+```html
+<div class="CardMedia CardMedia--auto">
+  <div class="CardMedia__canvas">
+    <a href="#" aria-hidden="true">
+      <img src="https://via.placeholder.com/300x200" alt="" />
+    </a>
+  </div>
+</div>
+```
+
+👉 Please note that the `aria-hidden="true"` attribute is used to hide the link from screen readers so the user is not
+confused by too many links in the Card.
+
+### The “Read More” Use Case
+
+For article previews or similar use cases, you may want to display a limited amount of text content with a “Read More”
+link. For optimum accessibility, you should only provide this in form of a text node, not a button or a link:
+
+```html
+<h4 class="CardTitle">
+  <a href="#" class="CardLink">Card title</a>
+</h4>
+<p><!-- … --></p>
+<!-- DON'T DO THIS -->
+<a href="#" class="link-primary link-underlined">Read more</a>
+<!-- This is correct -->
+<div class="link-primary link-underlined">Read more</div>
+```
+
+This way, the Card will only have a single accessible link which will be announced by screen readers.
+
+ℹ️ A big shout-out to [Ondřej Pohl][ondrej-pohl] for sharing many of these best practices!
```

Comment:
**suggestion (if-minor):** Can you also include the link to the article or presentation, if online, please?

Reply:
The WebExpo talk is inaccessible to public and I couldn't find the slides. But I found a recording of the same talk on YouTube: https://youtu.be/SdlgZFKxs40?feature=shared&t=1010

Reply:
Great, let's add it. 👍